File: QolmsYappli/QolmsYappli/Models/InnerModel/HealthAgeReportGraphItem.vb
<Serializable()>
Public NotInheritable Class HealthAgeReportGraphItem

#Region "Public Property"

    Public Property Title As String = String.Empty

    Public Property AxisMin As Decimal = Decimal.Zero

    Public Property AxisMax As Decimal = Decimal.Zero

    Public Property Label As String = "[]"

    Public Property TargetValue As String = "[]"

    Public Property Data As String = "[]"

#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="HealthAgeReportGraphItem" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
    End Sub

#End Region

End Class

File: QolmsJotoWebView/Models/InnerModels/HealthAgeReportGraphItem.cs
using System;

namespace MGF.QOLMS.QolmsJotoWebView
{
    [Serializable]
    public sealed class HealthAgeReportGraphItem
    {
        #region Public Property

        public string Title { get; set; } = string.Empty;

        public decimal AxisMin { get; set; } = decimal.Zero;

        public decimal AxisMax { get; set; } = decimal.Zero;

        public string Label { get; set; } = "[]";

        public string TargetValue { get; set; } = "[]";

        public string Data { get; set; } = "[]";

        #endregion

        #region Constructor

        /// <summary>
        /// <see cref="HealthAgeReportGraphItem" /> クラスの新しいインスタンスを初期化します。
        /// </summary>
        public HealthAgeReportGraphItem()
        {
        }

        #endregion
    }
}
